packages/eslint-plugin-pf-codemods/lib/helpers.js

```diff
@@ -4,37 +4,26 @@ function moveSpecifiers(
   specifiersToMove,
   fromPackage,
   toPackage,
-  messageAfterSpecifierPathChange,
-  messageAfterElementNameChange,
-  aliasSuffix
+  messageAfterSpecifierPathChange
 ) {
   return function (context) {
     const src = context.getSourceCode();
-    const importNames = specifiersToMove.map((nameToMove) => nameToMove.name);
     const { imports: fromPackageImports, exports: fromPackageExports } =
-      getFromPackage(context, fromPackage, importNames);
-    const allElements = getAllJSXElements(context);
-    const importSpecifiersToMove = fromPackageImports.filter((fromImport) => {
-      const foundElement = allElements.find(
-        (el) => el?.openingElement?.name?.name === fromImport.local?.name
-      );
-      const hasDataAttr = foundElement?.openingElement.attributes?.find(
-        (attr) => attr.name?.name === "data-codemods"
-      );
+      getFromPackage(context, fromPackage, specifiersToMove);
 
-      return (foundElement && !hasDataAttr) || !foundElement;
-    });
+    const getSpecifiersToMove = (specifiers) =>
+      specifiers.filter((specifier) => {
+        const comments = src.getCommentsAfter(specifier);
```

Comment:
can we add some tests that test `/* data-codemods */` both for valid (places where we don't want to operate on them if they exist) and invalid (like we have a `/* data-codemods */` on a different specifier and it doesn't prevent the codemod from working on the component specifier we are working with)?